CMakeLists.txt

```
@@ -131,6 +131,25 @@ option(
   "build the finite difference based space charge solver, requires PETSc & PkgConfig"
   OFF)
 
+# OpenPMD
+find_package(openPMD CONFIG)
+
+# if not found, fetch it
+if(NOT openPMD_FOUND)
+  include(FetchContent)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+  set(openPMD_BUILD_CLI_TOOLS OFF)
+  set(openPMD_BUILD_EXAMPLES OFF)
```

Maybe more usable for the superbuild is:
```suggestion
  set(openPMD_BUILD_SHARED_LIBS OFF)
  set(openPMD_INSTALL OFF)
```
that way you don't need to install and find a shared lib for openPMD.

---

Doc update coming here: https://github.com/openPMD/openPMD-api/pull/1325